pkg/systemd/host.js

```diff
@@ -195,6 +200,17 @@ function ServerTime() {
                 });
     };
 
+    self.ntp_updated = function ntp_updated(path, iface, member, args) {
+        if (!self.ntp_waiting_promise)
+            return;
+        if (args[1].NTP && self.ntp_waiting_value === args[1].NTP.v) {
+            self.ntp_waiting_promise.resolve();
+            self.ntp_waiting_promise = null;
+        } else {
+            console.log("Incorrect value of PropertiesChanged value");
```

Comment:
This should be raised to a warning IMHO. Also, the promise should still be resolved or rejected here. I'm not entirely sure which one is "more correct", but as we don't really depend on the reported value, we could just resolve the promise anyway.

But more importantly, this will log the message if any other property than `NTP` changes. Please restructure the if  condition to only handle that property.